### ISSUE TITLE
Multiple tags with the same name

### ISSUE THREAD

**Issue by chenryoab** [Altran-PT-GDC/Robot-Framework-SOAP-Library#13]
Hello,

Using the library, working fine, but i have only one problem with the 'Edit XML Request'. Our XML file has multiple tags with the same name, example :
     
    <ContactMethod>
                  <local_lastUpdateDate>2019-06-03 11:23:40.265</local_lastUpdateDate>
                  <contactMethodType>EmailAddress</contactMethodType>
                  <ContactMethodStatus>
                     <startDate>2019-06-03T11:23:40.255+02:00</startDate>
                  </ContactMethodStatus>
                  <ContactMethodQuality>
                     <undeliveredReasonType>1010</undeliveredReasonType>
                  </ContactMethodQuality>
    </ContactMethod>
    <ContactMethod>
                  <local_lastUpdateDate>2019-06-03 11:23:40.237</local_lastUpdateDate>
                  <contactMethodType>PhoneNumber</contactMethodType>
                  <ContactMethodStatus>
                     <startDate>2019-06-03T11:23:40.218+02:00</startDate>
                  </ContactMethodStatus>
                  <ContactMethodQuality>
                     <undeliveredReasonType>1010</undeliveredReasonType>
                  </ContactMethodQuality>
      </ContactMethod>

I want to update both of tags "undeliveredReasonType", so i tried to update the xml with the dictionary : 

 ${new_value_dict}    Create Dictionary          undeliveredReasonType2=1010         undeliveredReasonType=1030                      

But its updating only the first tag, not the second one. 

Any idea how to update both please?

Thanks

**Comment by samuelpcabral** [comment]
@chenryoab You need update both tags with same value? ou different values?

**Comment by samuelpcabral** [comment]
@chenryoab Both approaches were implemented in version 0.5.

Please run: pip install robotframework-soaplibrary --upgrade

Then to update all tags:

```
${new_value_dict}    Create Dictionary    undeliveredReasonType=1030
${xml_edited}    Edit XML Request    ${CURDIR}${/}request.xml    ${new_value_dict}    New_Request
```

Or to update only the second:

```
${new_value_dict}    Create Dictionary    undeliveredReasonType=1030
${xml_edited}    Edit XML Request    ${CURDIR}${/}request.xml    ${new_value_dict}    New_Request    repeated_tags=1
```
